lib/matplotlib/tests/test_mlab.py

```diff
@@ -339,6 +339,14 @@ def test_rec2csv_bad_shape_ValueError(self):
         # the bad recarray should trigger a ValueError for having ndim > 1.
         assert_raises(ValueError, mlab.rec2csv, bad, self.fd)
 
+    def test_csv2rec_names_with_comments(self):
+        self.fd.seek(0)
```

Comment:
this initial seek is unnecessary. nosetest calls self.setUp() and self.tearDown() before and after each test method.